jupyterhub/values.yaml

```diff
@@ -228,7 +228,7 @@ singleuser:
 
 scheduling:
   userScheduler:
-    enabled: false
+    enabled: true
     replicas: 1
```

Comment:
```suggestion
    replicas: 2
```

I think we should also bump the replicas to two so one can be moved around without loosing availability to schedule pods, they are very cheap on resource requests.

Reply:
Note that we do have a pdb eabled by default with 1 minAvailable, so right now that could bug people.

Reply:
Yeah thinking more about this, introducing the user-scheduler by default should go hand in hand with this bump as it could result in unexpected creation of a pod that stops scale downs for core nodes or user nodes if they are not tainted to disallow the scheduler to show up there.

Reply:
Commit this and merge? =)